orbit-2-livedata/README.md

```diff
@@ -0,0 +1,47 @@
+# Orbit 2 Live Data plugin
+
+The Live Data plugin provides:
+
+- [Container](../orbit-2-core/src/main/java/com/babylon/orbit2/Container.kt)
+  extensions for LiveData streams.
```

Comment:
again tempted to suggest merging the description of the extension properties into this